openhcl/openhcl_boot/src/main.rs

```diff
@@ -225,6 +225,11 @@ fn build_kernel_command_line(
         )?;
     }
 
+    // Only when explicitly supported by Host.
+    if partition_info.nvme_keepalive && !partition_info.vtl2_pool_memory.is_empty() {
```

Comment:
We should pass this via DT to usermode, the env args are for test/unstable features.

Reply:
Although if we want to keep it as unstable (env arg only), lets file a cleanup issue to track cleaning this up.

Reply:
okay, after your change I thought you wanted to keep it as command-line

Reply:
After further thinking I suggest to keep it as an env var for now and clanup later.

Reply:
https://github.com/microsoft/openvmm/issues/517